docs/src/pages/components/placeholder/examples/PlaceholderIsLoadedExample.tsx

```diff
@@ -0,0 +1,5 @@
+import { Placeholder } from '@aws-amplify/ui-react';
+
+export const PlaceholderIsLoadedExample = () => {
+  return <Placeholder isLoaded />;
```

Comment:
nit: `<Placeholder isLoading="true" />;` I think we agreed a while ago to be explicit when demonstrating these props